tests/phpunit/api/v4/Entity/CaseTest.php

```diff
@@ -114,6 +115,24 @@ public function testGetStatusIdPerCaseType(): void {
     $this->assertEquals(['Closed', 'Testing'], $options);
   }
 
+  public function testCaseTypeDefinition(): void {
+    $caseType = $this->createTestRecord('CaseType', [
+      'title' => 'Test Case Type',
+      'name' => 'test_case_type3',
+      'definition' => [
+        'statuses' => ['Testing', 'Closed'],
+      ],
```

Comment:
@jaapjansma unless I'm missing something, this test will pass with or without the patch, because you're putting the  definition into the SQL table. The case your patch solved was when the definition was *not* in the SQL and had to be retrieved from xml by `$field['output_formatters']`.

Reply:
Ah I see what you mean. I will change the test in a follow up PR as this one just got merged